src/template.yml

```diff
@@ -42,6 +42,9 @@ Parameters:
   DeploymentAccountName:
     Type: String
     Default: ""
+    AllowedPattern: "[\\s\\S]"
```

Comment:
`[\s\S]*`

Missing `*`

Reply:
as per: https://docs.aws.amazon.com/organizations/latest/APIReference/API_Account.html

Reply:
Oh my, I really need coffee ;)
I referenced the same article in the commit, haha.